base_layer/core/src/transactions/transaction_protocol/sender.rs

```diff
@@ -99,6 +104,9 @@ pub struct SingleRoundSenderData {
     pub metadata: TransactionMetadata,
     /// Plain text message to receiver
     pub message: String,
+    /// Dust collector enabled/disabled
+    #[serde(default)]
+    pub dust_collector_enabled: bool,
```

Comment:
Is this field something we want to be sending over the wire? It feels like it should be a private setting to each wallet.

Reply:
Agreed, had the wrong understanding here. The flag only needs to go from the sender to the base node for final validation, not to the receiver.